docs/runtime/transfer_learning.md

```diff
@@ -0,0 +1,197 @@
+# Transfer Learning
+
+Let's classify dogs and cats using the pre-trained model. It will be `mobilenet v2` in our case.
+
+## Preparing dataset
+The first step is conversion dataset (input data and expected outputs) to the binary format accepted by `ONE`. Note, that for many cases it can be achieved via [tf_dataset_converter](https://github.com/Samsung/ONE/tree/master/tools/generate_datafile/tf_dataset_converter).
+
+```python
+import os
+import tensorflow as tf
+import pathlib
+import numpy as np
+
+_URL = 'https://storage.googleapis.com/mledu-datasets/cats_and_dogs_filtered.zip'
+path_to_zip = tf.keras.utils.get_file('cats_and_dogs.zip', origin=_URL, extract=True)
+PATH = os.path.join(os.path.dirname(path_to_zip), 'cats_and_dogs_filtered')
+
+train_dir = os.path.join(PATH, 'train')
+validation_dir = os.path.join(PATH, 'validation')
+
+BATCH_SIZE = 32
+IMG_SIZE = (160, 160)
+
+def label_to_array(label):
+    arr = np.zeros(2, dtype=float)
+    arr[label] = 1.
+    tensor = tf.convert_to_tensor(arr, tf.float32)
+    return tensor
+
+train_dataset = tf.keras.preprocessing.image_dataset_from_directory(train_dir,
+                                                            shuffle=True,
+                                                            batch_size=BATCH_SIZE,
+                                                            image_size=IMG_SIZE)
+
+current_dir = pathlib.Path(__file__).parent
+with open(current_dir/'cats_and_dogs.input.bin', 'wb') as in_stream:
+    with open(current_dir/'cats_and_dogs.output.bin', 'wb') as out_stream:
+        for image,label in train_dataset:
+            in_stream.write(image.numpy().astype(np.float32).tobytes())
+            out_stream.write(label.numpy().astype(np.float32).tobytes())
+```
+
+## Feature extraction
```

Comment:
It's minor things. How about revisiting this name?

In the same reason in my old comments - https://github.com/Samsung/ONE/pull/13329#discussion_r1665180014. 

In my preference, 
```suggestion
## Prepare fine-tuning model 
```

or 
```suggestion
## Prepare model for fine tuning
```

Reply:
But it's just a suggestion, You could keep going with current version :smile:

Reply:
Sorry, I've missed it `Prepare model for fine-tuning` looks ok for me ;)